lib/imports.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const {getOriginal} = require('./originals-helper.js');
+
+/**
+ * Formats the information into the right format for import
+ *
+ * @param {string} identifier - the name of the method / function to get
+ * @param {string} type - the type of original of the identifier
+ * @param {string} mod - the name of the module to import from
+ *
+ * @return {string} the formatted import text
+ */
+function generateImportText(identifier, type, mod) {
+  const original = getOriginal(identifier, type, mod);
+  switch (type) {
+    case 'constructor':
+      return original;
+    case 'method':
+      return `${identifier} as ${original}`;
+    default:
+      throw new Error('not yet implemented');
+  }
+}
+
+/**
+ * Adds an import for the module to the list
+ *
+ * @param {string} mod - the name of the module to import from
+ * @param {string} identifier - the name of the method / function to get
+ * @param {string} type - the type of original of the identifier
+ * @param {Object} imports - the current list of imports
+ */
+function updateImports(mod, identifier, type, imports) {
+  const importText = generateImportText(identifier, type, mod);
+
+  // TODO(jackbsteinberg): update this for other call-requiring cases
+  // add original for `call` from `Reflect` upon first method call
```

Comment:
Is this TODO still accurate?

It seems like the case that's missing is what happens if Reflect already exists. For example an input file like

```js
Reflect.has({}, "prop");
const x = document.createElement('div');
```

or even

```js
Reflect.call(() => {});
const x = document.createElement('div');
```

But the TODO doesn't seem very related to those.

It might be good to add tests for those cases, although getting them right might want to wait until you do static and namespace methods.

Reply:
So I tried rephrasing this TODO but the language is definitely confusing, what I was trying to capture here is that there may be other `type`s that need to use `call` beyond `method`, like for instance `static`. Is that accurate?

I'll go ahead and fix this problem you mentioned as well.

Reply:
I guess in that case I'd remove it, since it's not a problem with the current code, just a task for future work.

Reply:
So if I'm not meant to prepare for the future by writing code future-proofing my code in ways that don't have to do with the current change, and I'm not meant to put comments in reminding myself of the future work I'm meant to do, how do I keep things from slipping through the cracks as I notice them?

Reply:
That's what I use the issue tracker for. In general code that gets merged to master should not be used as an issue tracker or to-do list; it should be something folks can read and understand in isolation as "what works now", not a work in progress.

Reply:
In general try to avoid TODO(person) because then you're using the code as an issue tracker. It does sometimes make sense to have TODO(ISSUE-URL), e.g. when you want to explain why something is wrong or missing in the code.